feat: add mainnet v1.0 update instructions (#1114)
axelarnetwork/axelar-docs: src/pages/resources/mainnet/upgrades/v1.0.mdx

# Mainnet upgrade: v1.0

import { Callout } from '/src/components/callout'

## About the v1.0 upgrade

> 

Reviewer: Mention what's being upgraded and to which specific version first (axelard v1.0.1 etc.). It's hidden at the bottom rn

> This release finalizes initial support for [Amplifier](/dev/amplifier/introduction).

Reviewer: ```suggestion
This release enables mainnet deployment for [Amplifier](/dev/amplifier/introduction).
```
reuse same text as the announcement

> Changes since 0.35.x:

Reviewer: Create a section header for changelog?

- Various fixes that improve stability
- New queries in the axelarnet module to query the IBC path for a chain and the chain by IBC path
- Added a new message type “BatchRequest”. In contrast to cosmos default behaviour, where the entire tx gets rolled back if there is an error in any of the contained message executions, this batch only rolls back the state for the specific messages that fail and keeps the rest
- Allow refundable messages to be batched
- Correctly pass message IDs between core and the new amplifier
- Add functionality to quickly (de)activate the connection between core and amplifier in case of emergencies

See all the changes in the [`axelar-core` release notes](https://github.com/axelarnetwork/axelar-core/releases).

> Upgrade height `14216700` at approximately **3:00PM UTC on August 27th, 2024** ([countdown](https://www.mintscan.io/axelar-testnet/block/14625900)) 

Reviewer: the mintscan link is to testnet instead of mainnet. Upgrade time is 10:30 am ET

<Callout type="warning" emoji="⚠️">

> As of v1.0.0, the `axelard` [binary releases](https://github.com/axelarnetwork/axelar-core/releases) require `GLIBC_2.34`, which is not present on Ubuntu 20.04.

Reviewer: ```suggestion
As of v1.0.1, the `axelard` [binary releases](https://github.com/axelarnetwork/axelar-core/releases) require `GLIBC_2.34`, which is not present on Ubuntu 20.04.
```
just in case people misunderstand this as upgrading to v1.0.0

# Mainnet upgrade: v1.0.1

import { Callout } from '/src/components/callout'

> ## About the `axelard` v1.0.1 upgrade

Reviewer: Can you mention each binary? axelard, vald and tofnd. We listed this out explicitly in previous instructions

- Allow refundable messages to be batched
- Correctly pass message IDs between core and the new amplifier
- Add functionality to quickly (de)activate the connection between core and amplifier in case of emergencies

See all the changes in the [`axelar-core` release notes](https://github.com/axelarnetwork/axelar-core/releases).

Upgrade height `14216700` at approximately **3:00PM UTC on August 27th, 2024** ([countdown](https://www.mintscan.io/axelar/block/14216700)) 

<Callout type="warning" emoji="⚠️">
As of v1.0.1, the `axelard` [binary releases](https://github.com/axelarnetwork/axelar-core/releases) require `GLIBC_2.34`, which is not present on Ubuntu 20.04.

You will need to compile `axelard` from source, or upgrade to Ubuntu 22.04.
</Callout>

<Callout type="warning" emoji="⚠️">
  If using the community node setup scripts, please update to the latest commit on `main` first.
  `wasmvm` lib will be auto-downloaded to `$AXELARD_HOME/lib` but you'll need to add it to `LD_LIBRARY_PATH` before running the script.
</Callout>

1. If you're a validator or have delegated to one, you can vote for the upgrade proposal, while it's active, via

```bash

> axelard tx gov vote 220 yes --from validator

Reviewer: ```suggestion
axelard tx gov vote [placeholder] yes --from validator
```
placeholder
